js/app/historyStore.js

```diff
@@ -369,6 +375,32 @@ var HistoryStore = new Lang.Class({
         }
         this.set_current_item(HistoryItem.HistoryItem.new_from_object(target_item));
     },
+
+    can_share_current_item: function () {
+        let item = this.get_current_item();
+        return (item.model &&
+                item.model instanceof Eknc.ContentObjectModel &&
+                item.model.original_uri !== null &&
+                item.model.original_uri !== "");
+    },
+
+    social_share: function (network) {
+        if (!this.can_share_current_item())
+          return;
+        let item = this.get_current_item();
+
+	let uri = encodeURIComponent (item.model.original_uri);
```

Comment:
The indentation is messed up in this entire block (from this line and on).

Reply:
Yeah, I just started using Builder last week